src/utils/ProteomicsLFQ.cpp

```diff
@@ -538,7 +538,8 @@ class ProteomicsLFQ :
           // Determine reference from data, otherwise a change in order of input files
           // leads to slightly different results
           const int reference_index(-1); // set no reference (determine from data)
-          Param ma_param = getParam_().copy("align_algorithm:", true);
+          Param ma_param = mat_param.copy("align_algorithm:", true);
```

Comment:
is this subsection naming in the same style as others of this tool?

Reply:
This is a sub*sub*section and is part of MapAlignerTreeGuided. I only expose MapAlignerTreeGuided settings  now (which include the MapAlignerID settings).